tests/handlebars-basics/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -20,12 +20,10 @@ printWidth: 80
 =====================================output=====================================
 <script id="entry-template" type="text/x-handlebars-template">
   <div class="entry">
-    <h1>
-      {{title}}
-    </h1>
+    <h1>{{title}}</h1>
```

Comment:
This shouldn’t have changed, should it?

Reply:
he must ran `yarn` , changed lock https://github.com/prettier/prettier/pull/6570#issuecomment-550266720

Reply:
I thought we fixed that problem?

Reply:
no, not fixable for now

Reply:
sorry, I mean he resintalled, not `yarn`,  maybe `npm install`?

Reply:
Reverted. Sorry, I didn't review the changed files before committing.